docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+ network.sh createChannel -c channelName
+ network.sh deployCC -l node
+```
+
+From inside the `test-network` directory, you can bring up the network by issuing
+the following command. You will experience problems if you try to run the script
+from another directory:
+```
+./network.sh up
+```
+
+This command creates a Fabric network that consists of two peer nodes, one
+ordering node, and an instance of the Fabric tools container. If the command
+completes successfully, you script will print out the following logs of the nodes
+being created.
+```
+
+Creating network "net_byfn" with the default driver
+Creating volume "net_orderer.example.com" with default driver
+Creating volume "net_peer0.org1.example.com" with default driver
+Creating volume "net_peer0.org2.example.com" with default driver
+Creating peer0.org1.example.com ... done
+Creating orderer.example.com    ... done
+Creating peer0.org2.example.com ... done
+Creating cli                    ... done
+CONTAINER ID        IMAGE                               COMMAND             CREATED             STATUS                  PORTS                    NAMES
+3d944d39fab3        hyperledger/fabric-tools:latest     "/bin/bash"         2 seconds ago       Up Less than a second                            cli
+280eaff2c840        hyperledger/fabric-peer:latest      "peer node start"   5 seconds ago       Up 1 second             0.0.0.0:9051->9051/tcp   peer0.org2.example.com
+0907470d42ee        hyperledger/fabric-orderer:latest   "orderer"           5 seconds ago       Up 1 second             0.0.0.0:7050->7050/tcp   orderer.example.com
+f1edf38980b5        hyperledger/fabric-peer:latest      "peer node start"   5 seconds ago       Up 2 seconds            0.0.0.0:7051->7051/tcp   peer0.org1.example.com
+```
+
+If you don't get this result, jump down to [Troubleshooting](#troubleshooting)
+for help on what might have gone wrong.
+
+### The components of the test network
+
+When the network running on your machine, you should take some time to examine
+what was created by the script. Run the following command to list all of Docker
+containers that are running on your machine. You should see the four nodes that
+were created by the `network.sh` script:
+```
+docker ps -a
+```
+
+Each node and user that interacts with a Fabric network needs to belong to an
+organization that is a network member. The group of organizations that are
+members of a Fabric network are often referred to as the consortium. The test
+network has two consortium members, `org1.example.com` and `org2.example.com`.
+The network also includes one orderer organization, `example.com`, who maintains
+the ordering service of the network.
+
+Peer are the fundamental components of any Fabric network. Peers store the
+blockchain ledger and validate transactions before they are committed to the
+ledger. Peers run the smart contracts that encode the business logic that is used
+to manage the assets on the blockchain ledger.
```

Comment:
Might be better saying something like "Because peers store the ledger and validate (etc -- however you want to describe what they do), they are fundamental components of any Fabric network." You might also want to say here that Org1 and Org2 own peers (whereas the orderer org does not). Therefore, you can think of Org1 and Org2 as peer organizations.

Reply:
I am going to leave as is to keep things simple.